File: Development_platforms/module-1/python/task-5.js
for (let i = 3; i < 100000; i++) {
	let isPrime = true;
	for (let k = 2; k < i; k++) {
		if (i % k == 0) {
			isPrime = false;
			break;
		}
	}
	if (isPrime) {
		console.log(i);
	}
}

File: Development_platforms/module-1/python/task-5.py

# Printe ut alle primtall mellom 3 og 100

for i in range(3,100):
    isPrime = True
    for k in range(2,i):
        if(i%k == 0):
            isPrime = False
            break
    if(isPrime):
        print(i)    
